styleguide.config.js

```diff
@@ -320,7 +320,7 @@ module.exports = {
   pagePerSection: true,
   showCode: true,
   showUsage: true,
-  serverPort: Number(process.env.PORT),
+  serverPort: 4040,
```

Comment:
could this be something like this instead? 
`serverPort: process.env.PORT || 4040`